examples/webgl_shaders_tonemapping.html

```diff
@@ -6,11 +6,19 @@
 		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 		<link type="text/css" rel="stylesheet" href="main.css">
 		<style>
-			.label {
-				background-color: black;
+			#labels {
 				position: absolute;
-				z-index: 100;
-				padding: 5px
+				display: grid;
```

Comment:
BTW: `grid` makes it very easy to perfectly center each label in its column:

https://raw.githack.com/Mugen87/three.js/dev54/examples/webgl_shaders_tonemapping.html
https://threejs.org/examples/webgl_shaders_tonemapping.html